game-core/src/main/java/tools/map/making/MapPropertiesMaker.java

```diff
@@ -398,27 +398,5 @@ private void handleCommandLineArgs(final String[] args) {
         log.severe("Not a decimal percentage: " + zoomString);
       }
     }
-    final String widthString = System.getProperty(ToolArguments.UNIT_WIDTH);
-    if (widthString != null && widthString.length() > 0) {
-      try {
-        final int unitWidth = Integer.parseInt(widthString);
-        mapProperties.setUnitsWidth(unitWidth);
-        mapProperties.setUnitsCounterOffsetWidth(unitWidth / 4);
-        log.info("Unit Width to use: " + unitWidth);
-      } catch (final Exception e) {
-        log.severe("Not an integer: " + widthString);
-      }
-    }
-    final String heightString = System.getProperty(ToolArguments.UNIT_HEIGHT);
-    if (heightString != null && heightString.length() > 0) {
-      try {
-        final int unitHeight = Integer.parseInt(heightString);
-        mapProperties.setUnitsHeight(unitHeight);
-        mapProperties.setUnitsCounterOffsetHeight(unitHeight);
-        log.info("Unit Height to use: " + unitHeight);
-      } catch (final Exception e) {
-        log.severe("Not an integer: " + heightString);
-      }
-    }
```

Comment:
This change isn't 100% clear to me either

Reply:
Likely due to unused properties and no longer need the catch, regardless the updates to this file were rolled-back to concentrate on cleanly removing the methods in `MapProperties.java`: e05930636